ft: BKTCLT-5 get bucket information

vrancurel opened the pull request:
There is currently no routine to exacly get the raft session ID
from the bucket name. It is possible to use getRaftInformations()
but it contains the state of the repd servers which contains the
raft ID so this is not very clear.

bert-e commented:
# Hello vrancurel,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/bucketclient/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue BKTCLT-5 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `8.1.2`


Please check the `Fix Version/s` of BKTCLT-5, or the target
branch of this pull request.

vrancurel committed:
ft: BKTCLT-5 get bucket information

There is currently no routine to exacly get the raft session ID
from the bucket name. It is possible to use getRaftInformations()
but it contains the state of the repd servers which contains the
raft ID so this is not very clear.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

vrancurel commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/8.1`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.10`
* `development/7.4`
* `stabilization/7.10.2`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/8.1`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.10`
* `development/7.4`
* `stabilization/7.10.2`



Please check the status of the associated issue BKTCLT-5.

Goodbye vrancurel.
